Doc/library/dataclasses.rst

```diff
@@ -356,7 +356,7 @@ Module-level decorators, classes, and functions
          def add_one(self):
              return self.x + 1
 
-.. function:: replace(instance, **changes)
+.. function:: replace(instance, /, **changes)
```

Comment:
Maybe add a versionchanged 3.9 markup to document this subtle change?

Reply:
It was not added in other 35 cases. If it is needed it should be added in all cases.